go.mod

```diff
@@ -33,6 +33,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.14.6
 )
 
+require github.com/openstack-k8s-operators/telemetry-operator/api v0.0.0-20230414131341-1a58e5589380
```

Comment:
Nit: it would be nice to have this grouped together with the rest of the `openstack-k8s-operators` in the `require` above

Reply:
ack